hbase-server/src/main/java/org/apache/hadoop/hbase/master/assignment/RegionStates.java

```diff
@@ -627,7 +627,21 @@ public boolean hasRegionsInTransition() {
 
   public boolean isRegionInTransition(final RegionInfo regionInfo) {
     final RegionStateNode node = regionInTransition.get(regionInfo);
-    return node != null ? node.isInTransition() : false;
```

Comment:
Does it make sense to delete the other calls to isInTransition in this class, I.e. in the method below? Seems redundant now

Reply:
Yes it does, agreed.